src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2478,10 +2479,12 @@ bool Eltwise::canFuse(const NodePtr& node) const {
             return false;
         }
 
-        for (const auto& originalInputPrecision : node->getOriginalInputPrecisions()) {
-            if (originalInputPrecision != ov::element::i32) {
-                return false;
-            }
+        if (!std::all_of(node->getOriginalInputPrecisions().begin(),
```

Comment:
Maybe to introduce utils::all_of_values (std::find(..., value) != end);

Reply:
Done